WordPressAuthenticator.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressAuthenticator"
-  s.version       = "1.27.0"
+  s.version       = "1.27.1-beta"
```

Comment:
`1.27.0` is the last release version, and we now leave the minor version for hotfixes, so this should probably be `1.28.0-beta.1`. (We drop the beta version in the host apps, i.e. WordPress will be `1.28.0-beta`.)

Reply:
Thank you!